PathsActivity.py

```diff
@@ -387,14 +385,15 @@ def _take_a_turn(self, nick):
         if nick == self.nick:
             self._game.its_my_turn()
         else:
+            self.set_player_on_toolbar(self,nick)
             self._game.its_their_turn(nick)
 
     def send_event(self, command, payload):
         """ Send event through the tube. """
-        if hasattr(self, 'chattube') and self.collab is not None:
+        if self.collab is not None:
             self.collab.post(dict(
-                command=command,
-                payload=payload
+                action = command,
+                new_text = payload
```

Comment:
I'm guessing you didn't notice, these keyword arguments shouldn't be spaced.

Reply:
I have corrected it in commit `766b774`. I'm not sure what functional change does this make? Or is it to make code uniform across all activities?

Reply:
Hi.  It is not a functional change, but a style change.  Style we use is to have no whitespace either side of = in keyboard arguments.  This helps us to immediately recognise it as not a variable assignment.  It showed up when we looked at the diff.

p.s. when you quote commits we can't hover or click on them, please don't wrap them in tick marks.  766b774.

Reply:
Oops, okay! :)